package.json

```diff
@@ -215,6 +215,7 @@
 		"lerna": "7.1.4",
 		"lint-staged": "10.0.1",
 		"make-dir": "3.0.0",
+		"md5": "^2.2.1",
```

Comment:
Just use the web crypto api, it's available in all browsers and in workers too. No md5